fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImpl.java

```diff
@@ -1492,6 +1494,48 @@ private List<ResultsetRowData> fillDatatableResultSetDataRows(final String sql)
         return resultsetDataRows;
     }
 
+    private List<ResultsetRowData> fillDatatableResultSetDataRows(final String sql, final List<ResultsetColumnHeaderData> columnHeaders) {
+        final List<ResultsetRowData> resultsetDataRows = new ArrayList<>();
+        final GenericResultsetData genericResultsetData = new GenericResultsetData(columnHeaders, null);
+
+        final SqlRowSet rowSet = jdbcTemplate.queryForRowSet(sql); // NOSONAR
+        final SqlRowSetMetaData rsmd = rowSet.getMetaData();
+
+        while (rowSet.next()) {
+            final List<Object> columnValues = new ArrayList<>();
+
+            for (int i = 0; i < rsmd.getColumnCount(); i++) {
+                final String columnName = rsmd.getColumnName(i + 1);
+                final String colType = genericResultsetData.getColTypeOfColumnNamed(columnName);
+                if ("DECIMAL".equalsIgnoreCase(colType)) {
+                    columnValues.add(rowSet.getBigDecimal(columnName));
+                } else if ("DATE".equalsIgnoreCase(colType)) {
+                    columnValues.add(rowSet.getDate(columnName));
+                } else if ("timestamp without time zone".equalsIgnoreCase(colType) // PostgreSQL
+                        || "datetime".equalsIgnoreCase(colType)) { // MariaDB
+                    Date dateVal = null;
+                    if (databaseTypeResolver.isMySQL()) {
+                        try {
+                            columnValues.add(rowSet.getObject(columnName));
+                        } catch (InvalidResultSetAccessException e) {
+                            LOG.error("ERROR: {}", e.getMessage());
```

Comment:
Bad pattern, let's just use the Logger#error overload which accepts an exception in its own, no need to only log the message.

Reply:
Done! It was removed because It was unnecesary